pytorch_toolkit/ote/ote/api.py

```diff
@@ -48,9 +48,9 @@ def test_args_parser(template_path):
     with open(template_path, 'r') as model_definition:
         config = yaml.safe_load(model_definition)
 
-        parser.add_argument('--test-ann-files', required=True,
+        parser.add_argument('--test-ann-files', required=False,
```

Comment:
what is the purpose of such change, is it for custom detector initialization?

Reply:
exactly